src/main/java/AdminTools/Punishment/TempBan.java

```diff
@@ -0,0 +1,140 @@
+package AdminTools.Punishment;
+
+import org.bukkit.BanList;
+import org.bukkit.Bukkit;
+import org.bukkit.ChatColor;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandExecutor;
+import org.bukkit.command.CommandSender;
+import org.bukkit.configuration.file.FileConfiguration;
+import org.bukkit.entity.Player;
+import org.bukkit.plugin.Plugin;
+import org.bukkit.scheduler.BukkitRunnable;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+public class TempBan implements CommandExecutor {
+    private final Plugin plugin;
+    private final Map<UUID, Long> temporaryBans = new HashMap<>();
+
+    private final String commandName;
+    protected String banMessageFormat;
+
+    public TempBan(Plugin plugin, String commandName, String banMessageFormat) {
+        this.plugin = plugin;
+        this.commandName = commandName;
+        this.banMessageFormat = banMessageFormat;
+        loadConfig();
+    }
+
+    private void loadConfig() {
+        plugin.saveDefaultConfig();
+        FileConfiguration config = plugin.getConfig();
+        banMessageFormat = ChatColor.translateAlternateColorCodes('&', config.getString("punishment.banMessageFormat"));
+        if (banMessageFormat == null) {
+            banMessageFormat = "&cYou have been temporarily banned for %time% due to %reason%.";
+            plugin.getLogger().warning("Ban message format not found in the configuration. Using default format.");
+        }
+    }
+
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        if (!(sender instanceof Player)) {
+            sender.sendMessage(ChatColor.RED + "This command can only be executed by a player.");
+            return false;
```

Comment:
`return false` will send the usage to the console which is not needed.
```suggestion
            return true;
```